nui-input/src/main/java/org/terasology/input/device/MouseDevice.java

```diff
@@ -34,19 +35,21 @@
     /**
      * @return The current position of the mouse in screen space
      */
-    Vector2i getMousePosition();
+    Vector2i getPosition();
 
     /**
      * @return The change in mouse position over the last update
      */
-    Vector2i getDelta();
+    Vector2d getDelta();
 
     /**
      * @param button
      * @return The current state of the given button
      */
     boolean isButtonDown(int button);
 
+    void update();
```

Comment:
Where does this come from? When is it called? Can we add some JavaDoc here, please?